plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -192,19 +196,15 @@ export class ExpressInstrumentation extends InstrumentationBase<
 
         // Rename the root http span in case we haven't done it already
         // once we reach the request handler
+        const rpcMetadata = getRPCMetadata(context.active());
         if (
           metadata.attributes[AttributeNames.EXPRESS_TYPE] ===
-          ExpressLayerType.REQUEST_HANDLER
+            ExpressLayerType.REQUEST_HANDLER &&
+          rpcMetadata?.type === RPCType.HTTP
```

Comment:
Odd spacing here, wonder if lint will fail?

Reply:
i actually don't know, i applied the eslint `--fix` option through my IDE and i passed lint locally